### DISCUSSION TITLE
Unicode support

### DISCUSSION THREAD

**Discussion by No-0n3** [cesanta/mongoose#2904] [Ask questions, get answers] [answered]
Hi!

Have search the docs but haven't found any confirmation. I assume that mongoose support unicode in both incoming and outgoing data as it's a webserver.

Would like to have a confirmation. Does it support unicode chars in incoming and outgoing data?

Br No-0n3

**Comment by scaprile** [comment, marked as answer]
Mongoose is not a webserver, it is a C library written in C to be linked against user code, there is no Unicode nor the lack of it in C. Mongoose doesn't care what is inside your data.
